### PR TITLE
1635 display sentence not only its name

### LINKED ISSUE #1635: display sentence, not only its name
On the proving page for a sentence, the sentence itself should be also displayed.
Also, in sentence selection menus, the sentence could be at least be displayed via mouse-over.

### PR DESCRIPTION
**Pull request by eugenk**
This shall fix #1635.

### CODE REVIEW

File: app/helpers/proofs_helper.rb [hunk 1]
```diff
@@ -16,4 +28,22 @@ def checked_axiom_selection_method
   def sine_value(field, default_value)
     resource.specific_axiom_selection.try(field) || default_value
   end
+
+  def proof_timeout_label(value)
```
Comment (eugenk) [inline app/helpers/proofs_helper.rb]: I don't see how this can be shortened.

File: app/views/proof_attempts/_sentences_list.html.haml [hunk 2]
```diff
@@ -4,6 +4,6 @@
       - if sentence.is_a?(GeneratedAxiom)
         %li= sentence
       - else
-        %li= link_to sentence, locid_for(sentence)
+        %li{class: 'has-tooltip', data: {toggle: 'tooltip', placement: 'top'}, title: sentence.text}= link_to sentence, locid_for(sentence)
```
Comment (eugenk) [inline app/views/proof_attempts/_sentences_list.html.haml]: 1. No, see b8966c530ef697cb70552695b6c5bd6da16fbe95
2. No, see b8966c530ef697cb70552695b6c5bd6da16fbe95
3. Okay.

File: app/helpers/proofs_helper.rb (hunk 1 above)
Comment (ebolloff) [inline app/helpers/proofs_helper.rb]: Maybe something like
```
if value >= 1.days
  some_method(days)
```
could be possible?

Comment (eugenk) [inline app/helpers/proofs_helper.rb]: This should be fixed now.

**Comment by ebolloff** [comment]
:+1:

**Pull request merged**